proto/ibc/core/channel/v1/tx.proto

```diff
@@ -358,13 +358,12 @@ message MsgChannelUpgradeOpenResponse {}
 message MsgChannelUpgradeTimeout {
   option (cosmos.msg.v1.signer) = "signer";
 
-  option (gogoproto.equal)           = false;
   option (gogoproto.goproto_getters) = false;
 
   string                    port_id                = 1;
   string                    channel_id             = 2;
   Channel                   counterparty_channel   = 3 [(gogoproto.nullable) = false];
-  ErrorReceipt              previous_error_receipt = 4 [(gogoproto.nullable) = false];
```

Comment:
made this change bc in the spec, [the error receipt can be nil](https://github.com/cosmos/ibc-go/pull/3905)